Gemfile

```diff
@@ -13,15 +13,9 @@ gem "rails", "~> 5.1.4"
 gem "pg", "~> 0.18"
 # Use Puma as the app server
 gem "puma", "~> 3.7"
-# Use SCSS for stylesheets
-gem "sass-rails", "~> 5.0"
-# Use Uglifier as compressor for JavaScript assets
-gem "uglifier", ">= 1.3.0"
-# See https://github.com/rails/execjs#readme for more supported runtimes
-# gem 'therubyracer', platforms: :ruby
-#
-# Turbolinks makes navigating your web application faster. Read more: https://github.com/turbolinks/turbolinks
-gem "turbolinks", "~> 5"
```

Comment:
You had mentioned wanting to keep a pjaxy thing around, I'm assuming webpacker will let us do that in a boring way?

Reply:
Yep, gonna add that back in after the scaffold is up